src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -60,216 +61,22 @@ import scala.collection.concurrent.TrieMap
 import scala.collection.mutable.ListBuffer
 import scala.concurrent.Await
 import scala.sys.process.Process
-import scala.util.Try
+import scala.util.{Random, Try}
 
 trait BeamHelper extends LazyLogging {
 
   protected val beamAsciiArt: String =
     """
-    |  ________
-    |  ___  __ )__________ _______ ___
-    |  __  __  |  _ \  __ `/_  __ `__ \
-    |  _  /_/ //  __/ /_/ /_  / / / / /
-    |  /_____/ \___/\__,_/ /_/ /_/ /_/
-    |
-    | _____________________________________
-    |
+      |  ________
+      |  ___  __ )__________ _______ ___
+      |  __  __  |  _ \  __ `/_  __ `__ \
+      |  _  /_/ //  __/ /_/ /_  / / / / /
+      |  /_____/ \___/\__,_/ /_/ /_/ /_/
+      |
+      | _____________________________________
+      |
     """.stripMargin
 
-  private def updateConfigForClusterUsing(
```

Comment:
@SHaaD94 what is the reason to move the codes?
Asking because during the bug investigations it will take sometime to realize that there were only moves of the lines

Reply:
it was just formatting of the code, can try to revert everything and just insert my changes

Reply:
@SHaaD94 yes, please.